src/addon/info.cpp

```diff
@@ -117,7 +117,7 @@ void addon_info::write(config& cfg) const
 	cfg["uploads"] = this->uploads;
 	cfg["type"] = get_addon_type_string(this->type);
 
-	for(const std::pair<std::string, addon_info_translation> &element : this->info_translations) {
+	for(std::pair<std::string, addon_info_translation> element : this->info_translations) {
```

Comment:
For reasons I don't quite understand only one compiler (clang 7) took fault to the original. Ideally this should be declared as `const auto& element` to avoid the copies and for readability (we all know the value type for `info_translations`, it's not necessary to specify it explicitly).

Reply:
Ok, now it's working well with that compiler.